nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -547,7 +496,6 @@ in
       programs.file-roller.enable = notExcluded pkgs.gnome.file-roller;
       programs.geary.enable = notExcluded pkgs.gnome.geary;
       programs.gnome-disks.enable = notExcluded pkgs.gnome.gnome-disk-utility;
-      programs.gnome-terminal.enable = notExcluded pkgs.gnome.gnome-terminal;
```

Comment:
This has the side effect of disabling `programs.bash.vteIntegration` and `programs.zsh.vteIntegration`, which should ideally still be enabled for gnome-console.